File: src/main/java/com/mpe85/grampa/input/InputPosition.java
package com.mpe85.grampa.input;

/**
 * Defines a position inside a text input (line and column)
 * 
 * @author mpe85
 *
 */
public class InputPosition {
	
	private final int line;
	private final int column;
	
	public InputPosition(
			final int line,
			final int column) {
		this.line = line;
		this.column = column;
	}
	
	public int getLine() {
		return line;
	}
	
	public int getColumn() {
		return column;
	}
	
}

File: src/main/kotlin/com/mpe85/grampa/input/InputPosition.kt
package com.mpe85.grampa.input

/**
 * A position inside a text input (line and column).
 *
 * @author mpe85
 *
 * @property[line] The line number
 * @property[column] The column number
 */
data class InputPosition(val line: Int, val column: Int)
